basic_integral/basic_integral.cpp

```diff
@@ -0,0 +1,24 @@
+#include <iostream>
+#include <math.h>
+
+using namespace std;
+
+int main(){
+    cout << "Iterations ?";
+    int n;
```

Comment:
Isn't it nice to have all declarations at the beginning of the function? Any reason for not doing so?